gitversion.go

```diff
@@ -105,13 +153,60 @@ func main() {
 					Name:  "patch",
 					Usage: "bump the patch version",
 					Action: func(c *cli.Context) error {
-						if err := BumpPatch(prefix); err != nil {
-							fmt.Fprintf(os.Stderr, "Error: %v", err)
+						if err := Bump(prefix, PATCH); err != nil {
+							fmt.Fprint(os.Stderr, "Error: ")
 							return err
 						}
 						return nil
 					},
 				},
+				{
+					Name:  "minor",
+					Usage: "bump the minor version",
+					Action: func(c *cli.Context) error {
+						if err := Bump(prefix, MINOR); err != nil {
+							fmt.Fprint(os.Stderr, "Error: ")
+							return err
+						}
+						return nil
+					},
+				},
+				{
+					Name:  "major",
+					Usage: "bump the major version",
+					Action: func(c *cli.Context) error {
+						if err := Bump(prefix, MAJOR); err != nil {
+							fmt.Fprint(os.Stderr, "Error: ")
+							return err
+						}
+						return nil
+					},
+				},
+				{
+					Name:  "auto",
+					Usage: "bump the version specified in the last commit",
+					Action: func(c *cli.Context) error {
+						if err := Bump(prefix, AUTO); err != nil {
+							fmt.Fprint(os.Stderr, "Error: ")
+							return err
+						}
+						return nil
+					},
+				},
+			},
+		},
+		{
+			Name:    "show",
+			Aliases: []string{"s"},
```

Comment:
Perhaps rename to "latest" or add as a subcommand. As a new user I would expect "show" to list all versions.